examples/todo/test/helpers.ts

```diff
@@ -30,12 +33,42 @@ import {Todo} from '../src/models/index';
  * @param todo A partial (or complete) Todo object.
  */
 export function givenTodo(todo?: Partial<Todo>) {
-  return Object.assign(
-    new Todo({
+  const data = Object.assign(
+    {
       title: 'do a thing',
       desc: 'There are some things that need doing',
       isComplete: false,
-    }),
+    },
     todo,
   );
+  return new Todo(data);
+}
+
+export const aLocation = {
+  address: '1 New Orchard Road, Armonk, 10504',
+  geopoint: <GeoPoint>{y: 41.109653, x: -73.72467},
+  get geostring() {
+    // tslint:disable-next-line:no-invalid-this
+    return `${this.geopoint.y},${this.geopoint.x}`;
+  },
+};
+
+const GEO_CODER_CONFIG = require('../src/datasources/geocoder.datasource.json');
+
+export function getProxiedGeoCoderConfig(proxy: HttpCachingProxy) {
+  const config = Object.assign({}, GEO_CODER_CONFIG);
```

Comment:
Your code snippet won't work because `Object.assign` performs a shallow merge (only top-level properties are merged) and`GEO_CODER_CONFIG` already contains `options`:

```json5
{
  // ...
  "options": {
    "headers": {
      "accept": "application/json",
      "content-type": "application/json"
    }
  },
  // ...
}
```

To avoid future confusion, I'll rework this code to use [`lodash.merge`](https://lodash.com/docs#merge).